docs/project_governance.rst

```diff
@@ -26,10 +97,32 @@ Steering Committee
 #. The Steering Committee votes on proposals and decides the allowed reviewers of
    `GitHub pull requests <https://github.com/ausseabed/qax/pulls>`_.
 
-Decision Process
-----------------
 
-#. A proposal is created as a `GitHub ticket <https://github.com/ausseabed/qax/issues>`_
+Project Work
+------------
+
+Definition and process
+^^^^^^^^^^^^^^^^^^^^^^
+
+A proposal can be classified into two categories: minor and major proposals.
+
+Some examples of minor proposals are:
+
+* Updates to Documentation
+* Bug fixes
+
+Some examples of major proposals are:
+
+* Any change involving a significant amount of new code
+* Changes that break backward compatibility
+
+
+Process
+^^^^^^^
+
+The following process is to be undertaken for all major proposals.
+
+#. A proposal is created as a `GitHub issue <https://github.com/ausseabed/qax/issues>`_
    for discussion and voting by any interested party (not just the committee members).
```

Comment:
We should mark it with a 'proposal' label to avoid being confused with the other tickets.